src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1519,4 +1519,13 @@
     <value>and all other platforms</value>
     <comment>This call site is reachable on: 'windows' 10.0.2000 and later, and all other platforms</comment>
   </data>
+  <data name="PreferHashDataOverComputeHashAnalyzerTitle" xml:space="preserve">
+    <value>Prefer static 'HashData' method over 'ComputeHash'</value>
+  </data>
+  <data name="PreferHashDataOverComputeHashAnalyzerDescription" xml:space="preserve">
+    <value>It is simpler to use the static 'HashData' method over 'ComputeHash'.</value>
```

Comment:
I think this description should also give a short explanation of _why_ it is simpler.